README.md

```diff
@@ -201,9 +203,44 @@ All sending errors are handled by the `onError` callback.
 
 All sending errors are handled by the `onError` callback.
 
+## Dats Mock
+
+Dats exports his mock, you can use it as follow:
+
+```ts
+import ClientMock from '@immobiliare/dats/mock';
+
+const host = new URL(`udp://127.0.0.1:8232`);
+const namespace = 'ns1';
+const client = new ClientMock({ host, namespace });
+
+client.gauge('some.metric', 100);
+client.set('some.metric', 100);
+// metrics is an array with all metrics send
+console.log(client.metrics);
+/* stdout:
+    [
+        'ns1.some.metric:100|g',
+        'ns1.some.metric:100|s',
+    ]
+*/
+// Check if a metric is in the metrics array
+client.includes('ns1.some.metric:100|s'); // -> true
```

Comment:
Now that I see it, I'm thinking if something like `client.sent("abc")` is more correct 🤔

Reply:
Mm, I like it. But my doubt is: do you think it is too similar to `send`?

Reply:
You got a point, we can use the verb like in most assertion libraries, something like `didSend`, `hasSend`. It'll make the test more readable!